test/modules/standard/FileSystem/lydia/isMount/symlinkToMount.prediff

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Note: I tried using a cleanfiles for this test, but it was failing to
```

Comment:
You could create and remove the symlinks internally to the chpl code (using FileSystem.symlink and FileSystem.remove) to get rid of the .preexec and .prediff files that create and delete them.  This is not required for this PR, but just an idea to clean this up.